ouroboros-consensus-cardano/src/shelley/Ouroboros/Consensus/Shelley/Ledger/Query.hs

```diff
@@ -774,6 +790,8 @@ encodeShelleyQuery query = case query of
       CBOR.encodeListLen 4 <> CBOR.encodeWord8 27 <> toCBOR coldCreds <> toCBOR hotCreds <> LC.toEraCBOR @era statuses
     GetFilteredVoteDelegatees stakeCreds ->
       CBOR.encodeListLen 2 <> CBOR.encodeWord8 28 <> LC.toEraCBOR @era stakeCreds
+    GetAccountState nes ->
+      CBOR.encodeListLen 2 <> CBOR.encodeWord8 29 <> LC.toEraCBOR @era nes
 
 decodeShelleyQuery ::
```

Comment:
I can't comment on unchanged lines, but you are missing a decoder here.

Reply:
Thanks!